http-clients/src/main/java/org/triplea/http/client/HttpClient.java

```diff
@@ -36,7 +35,7 @@
 @AllArgsConstructor
 public class HttpClient<ClientTypeT> implements Supplier<ClientTypeT> {
 
-  private static final Decoder gsonDecoder = new GsonDecoder();
+  private static final Decoder gsonDecoder = JsonDecoder.gsonDecoder();
```

Comment:
A custom JSON decoder was needed (fun) for Instant deserialization. The server encodes instant into an epoch second plus epoch second, which seems to not be natively supported.